pkgs/by-name/ju/juce/package.nix

```diff
@@ -81,6 +93,26 @@ stdenv.mkDerivation (finalAttrs: {
 
   propagatedBuildInputs = [ fontconfig ];
 
+  cmakeFlags = [ "-DJUCE_BUILD_EXTRAS=ON" ];
+
+  postInstall = ''
+    cp extras/Projucer/Projucer_artefacts/Release/Projucer $out/bin/Projucer
```

Comment:
We should probably add a symlink to projucer so that like most linux tools are only lowercased.